espresso-server/app/build.gradle

```diff
@@ -27,8 +27,9 @@ dependencies {
     compile 'com.android.support:appcompat-v7:25.0.0'
     testCompile 'junit:junit:4.12'
     compile 'com.android.support.constraint:constraint-layout:1.0.2'
-    compile 'org.nanohttpd:nanohttpd-webserver:2.3.1'
-    compile 'com.google.code.gson:gson:2.2.4'
+
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.2'
     androidTestCompile 'com.android.support.test:runner:0.5'
+    androidTestCompile 'org.nanohttpd:nanohttpd-webserver:2.3.1'
+    androidTestCompile 'com.google.code.gson:gson:2.2.4'
```

Comment:
I need Espresso, NanoHTTPD and GSON to be `compile` dependencies. The code for the espresso server will reside in the main code directory. The only thing in the `androidTest` directory is one test that imports the server and starts it and does nothing else.

I can change this myself though, this doesn't need to hold up merging it.